marklogic-data-hub-central/ui/e2e/cypress/integration/explore/verifyConceptsInExplore.spec.tsx

```diff
@@ -65,24 +60,21 @@ describe("Concepts", () => {
     cy.log("**Turn OFF concepts toggle**");
     graphView.getConceptToggle().scrollIntoView().trigger("mouseover").click();
 
-    cy.wait(3000);
+    cy.wait(4000);
 
     cy.log("**Verify Kettle concept node is not visible in the canvas anymore**");
     graphExplore.focusNode(ExploreGraphNodes.CONCEPT_KETTLE);
     graphExplore.getPositionsOfNodes(ExploreGraphNodes.CONCEPT_KETTLE).then((nodePositions: any) => {
       let kettleCoordinates: any = nodePositions[ExploreGraphNodes.CONCEPT_KETTLE];
-      const canvas = graphExplore.getGraphVisCanvas();
-
-      //Click on node to open side panel
-      canvas.click(kettleCoordinates.x, kettleCoordinates.y, {force: true});
-      canvas.click(kettleCoordinates.x, kettleCoordinates.y, {force: true});
-      graphExploreSidePanel.getSidePanel().should("not.exist");
+      //it should not exist because the leaf node was collapsed
+      cy.log("**Kettle coordinates should not exist because it was collapsed**");
+      expect(kettleCoordinates).to.be.undefined;
     });
 
     cy.log("**Turn ON concepts toggle**");
     graphView.getConceptToggle().scrollIntoView().trigger("mouseover").click();
 
-    cy.wait(3000);
```

Comment:
@csilva-ml Can please not use implicit waits

Reply:
Hey @ngodugu-marklogic , this wait was already here, I have only increased it one second. As you may know, canvas is not a very friendly web element for automation to interact with (this is why we click using coordinates and not specific elements). We need to wait for the graph to get stabilized, and there's no timeout I can increase for that. I cannot grab the elements inside the canvas so that's why I do not know when it has stopped animating. If you have any suggestion we can work on them for all the graph tests!

Reply:
My concern here is that we will have to again work on them when they run on nightly, as the wait times may vary on VMs. There will be rework again. Is there any API or an element we can wait for?

Reply:
@csilva-ml Please create a ticket to investigate the best way to validate graphs by not using waits. We can post a question online to cypress community and get some help. Also we can also try writing a polling function.